tools/scoring/tests/dimensions/entity_identification_test.py

```diff
@@ -17,34 +17,178 @@
 from absl.testing import absltest
 
 from score.dimensions.entity_identification import EntityIdentification
-from score.constants import FileTypes
+from score.constants import FileTypes, DimensionCategories
+from score.parse_config import ParseConfig
+
+from validate import handler as validator
+from validate.generate_universe import BuildUniverse
+
+import copy
 
 PROPOSED, SOLUTION = FileTypes
+SIMPLE, COMPLEX = DimensionCategories
 
 
 class EntityIdentificationTest(absltest.TestCase):
+  def _prepare_dimension_argument(self, *, entity_type, proposed_path,
+                                  solution_path):
+    """Prepare argument for direct invocation of a dimension for purposes of
+    testing (i.e. mimic parse_config.py).
+
+    NOTE: this uses the simplified universe. If the test data references types
+    which are not contained therein, results will differ from those which use
+    the full universe because entities with missing types are skipped!
+
+      Arguments:
+        entity_type: the category of the dimension. (Literal[SIMPLE, COMPLEX])
+        proposed_path: the path to the proposed YAML file
+        solution_path: the path to the solution YAML file
+
+      Returns:
+        The appropriate value for the dimension's singular named argument"""
+
+    universe = BuildUniverse(use_simplified_universe=True)
+    proposed_config = validator.Deserialize([proposed_path])[0]
+    solution_config = validator.Deserialize([solution_path])[0]
+    deserialized_files = {PROPOSED: proposed_config, SOLUTION: solution_config}
+
+    deserialized_files_appended = ParseConfig._append_types(  # pylint: disable=protected-access
```

Comment:
make public please remove `_` and add docstring

Reply:
It has a docstring.